Response/DatatableQueryBuilder.php

```diff
@@ -633,7 +633,7 @@ private function getMetadata($entityName)
      */
     private function getTableName(ClassMetadata $metadata)
```

Comment:
Nice. But this is no longer the name of the table. Maybe "shortEntityName" or something similar

``` php
getShortEntityName()
```